modules/base/rendering/pointcloud/renderablepointcloud.cpp

```diff
@@ -199,6 +203,29 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
+    constexpr openspace::properties::Property::PropertyInfo UseOrientationDataInfo = {
+        "UseOrientationData",
+        "Use Orientation Data",
+        "Include the orietation data in the datraset when rendering the poitns, if there "
+        "is any. To see the rotation, you also need to set the \"Orientation Render "
+        "Option\" to \"Fixed Rotation\"",
+        openspace::properties::Property::Visibility::AdvancedUser
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo OrientationRenderOptionInfo = {
+        "OrientationRenderOption",
+        "Orientation Render Option",
+        "Controls how the planes for the points will be oriented. \"Camera View "
+        "Direction\" rotates the points so that the plane is orthogonal to the viewing "
+        "direction of the camera (useful for planar displays), and \"Camera Position "
+        "Normal\" rotates the points towards the position of the "
+        "camera (useful for spherical displays, like dome theatres). In both these cases "
+        "the points will be billboarded towards the camera. In contrast, \"Fixed "
+        "Rotation\" does not rotate the points at all based on the camera and should be "
+        "used when the dataset contains orientation information for the points.",
```

Comment:
```suggestion
        "used when the dataset contains orientation information for the points",
```

I don't think we have periods at the end of the descriptions elsewhere

Reply:
Yes, we do.  But it is mixed all over in the code base 

IIRC, when created the RenderablePointsCloud renderable, I was asked to add points to the end.. 😅 

But we should probably decide on a convention to use.